projects/plugins/videopress/tests/e2e/package.json

```
@@ -1,5 +1,5 @@
 {
-	"name": "jetpack-starter-e2e-tests",
+	"name": "videopress-e2e-tests",
```

It looks like other e2e tests have `jetpack-` before the name of the test, what do you think if we change it to:

```suggestion
	"name": "jetpack-videopress-e2e-tests",
```
I don't have much experience with this repo but thought I'd mention it. Let me know, thanks!

---

@geriux good idea. I have committed the rename to align with existing convention. Let me know if you think this is ready for a Jetpack Crew team member review. 🙇🏻

---

Awesome! Thank you! 

> Let me know if you think this is ready for a Jetpack Crew team member review. 🙇🏻
Yup 👍